src/environmentd/tests/auth.rs

```diff
@@ -952,7 +952,7 @@ fn test_auth_base() {
         roles,
         role_rx,
         now.clone(),
-        1_000,
+        2_000,
```

Comment:
```suggestion
        1_000,
```

This doesn't need to change, it's not used.